k8s/operators/src/diskpool/crd.rs

```diff
@@ -66,11 +69,26 @@ pub enum PoolState {
     Error,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq, JsonSchema)]
+#[non_exhaustive]
+pub enum PoolStatus {
```

Comment:
missing doc comment for PoolStatus type.

Reply:
Added